admin/src/main/resources/io/buoyant/admin/spec/test-main.js

```diff
@@ -0,0 +1,54 @@
+var allTestFiles = []
+var TEST_REGEXP = /(spec|test)\.js$/i
+
+// Get a list of all the test files to include
+Object.keys(window.__karma__.files).forEach(function (file) {
+  if (TEST_REGEXP.test(file)) {
+    // Normalize paths to RequireJS module names.
+    // If you require sub-dependencies of test files to be loaded as-is (requiring file extension)
+    // then do not normalize the paths
+    var normalizedTestModule = file.replace(/^\/base\/|\.js$/g, '')
+    allTestFiles.push(normalizedTestModule)
+  }
+})
+
+require.config({
+  // Karma serves files under /base, which is the basePath from your config file
+  baseUrl: '/base',
```

Comment:
I think this is where karma likes to load its assets from:

http://karma-runner.github.io/1.0/config/files.html
```
By default all assets are served at http://localhost:[PORT]/base/
...
Notice the base in the URL, it is a reference to your basePath. You do not need to replace or provide your own base.
```

So if our basePath was "/foo" karma would serve from "/base/foo" - but I am not sure we want to change the /base part. It loads our assets with that /base/ in the url, e.g. http://localhost:9876/base/js2/lib/lodash.min.js with relative requires.

Looks like I can get around this by doing something like
```
proxies: {
  '/': '/base',
},
```
but we don't rely on the urls anyway... so it doesn't matter if we are using /base. What do you think?